techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st

```diff
@@ -37,7 +37,7 @@ bundle agent rudder_system_update_campaign_&RudderUniqueID& {
       "pass1"     expression => "any";
 
     pass2::
-      "call"      expression => usemodule("system_update.py", "--package_manager=${package_module_knowledge.platform_default} --campaign_id=${campaign_id} --start=${start} --end=${end} --node_id=${g.uuid} --agent_schedule=${g.agent_run_interval} ${reboot_opt}"),
+      "call"      expression => usemodule("system_update.py", "--package_manager=${package_module_knowledge.platform_default} --campaign_id=${campaign_name} --start=${start} --end=${end} --node_id=${g.uuid} --agent_schedule=${g.agent_run_interval} ${reboot_opt}"),
```

Comment:
isn't the id suppoed to be a uuid or a canonified string? If not we nee to modify the script too to canonify it before using it in filenames.

Reply:
an uuid

Reply:
with _tiret du six_?

Reply:
the famous "tiret du six" aka a dash

Reply:
I kep the canonfied value where it should (in cfengine classes), I checked in the module that it was just used as a filename